config/src/content/jcr_root/apps/_rep_policy.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
+          jcr:primaryType="rep:ACL">
+  <clientlibs-access
+      jcr:primaryType="rep:GrantACE"
+      rep:principalName="clientlibs-service"
+      rep:privileges="{Name}[jcr:read]"/>
```

Comment:
this is pretty dramatically changing the OOTB service user's permissions, we could at least limit this to a glob of `*/clientlibs/*`. it would probably result in needing to change the logic in place in the aggregator right now from getting the component resource and then doing `getChild()` to just doing `resolver.getResource(resourceTypePath + primaryPath)` kind of thing. otherwise we may have to consider a new service user or something to that effect.